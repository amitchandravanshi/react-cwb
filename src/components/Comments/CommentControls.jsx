import React from "react";
import { withStyles } from "@material-ui/core/styles";
import CardActions from "@material-ui/core/CardActions";
import IconButton from "@material-ui/core/IconButton";
import red from "@material-ui/core/colors/red";

const styles = theme => ({
  actions: {
    display: "flex"
  },
  avatar: {
    backgroundColor: red[500]
  },
  cardActionBtn: {
    height: `${theme.spacing * 3 + 4}px`,
    width: `${theme.spacing * 3 + 4}px`
  },
  cardActionIcon: {
    color: theme.palette.primary.pink
  }
});

const CommentControls = props => {
  const { classes } = props;
  return (
    <CardActions className={classes.actions} disableActionSpacing>
      <IconButton className={classes.cardActionBtn} />{" "}
    </CardActions>
  );
};

export default withStyles(styles)(CommentControls);
