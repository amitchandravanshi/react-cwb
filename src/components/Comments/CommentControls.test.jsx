import React from "react";
import { shallow, mount } from "enzyme";
import toJson from "enzyme-to-json";
import CommentControls from "./CommentControls";
import { wrap } from "module";

const props = {};
describe("<CommentControls>", () => {
  it("<CommentControls> snapshort ", () => {
    let wrapper = shallow(<CommentControls {...props} />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
  it("<CommentControls> classname should null ", () => {
    let wrapper = mount(<CommentControls {...props} />);
    console.log(wrapper.instance().props.classes);
    expect(wrapper.instance().props.classes).toBeUndefined();
  });
});
