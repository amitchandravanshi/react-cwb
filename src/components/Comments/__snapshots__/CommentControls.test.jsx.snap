// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CommentControls>  <CommentControls> snapshot  1`] = `
<CommentControls
  classes={
    Object {
      "actions": "CommentControls-actions-1",
      "avatar": "CommentControls-avatar-2",
      "cardActionBtn": "CommentControls-cardActionBtn-3",
      "cardActionIcon": "CommentControls-cardActionIcon-4",
    }
  }
/>
`;
